tests/infra_test.go

```diff
@@ -767,25 +767,13 @@ var _ = Describe("[Serial]Infrastructure", func() {
 		})
 
 		It("[test_id:4146]should include VMI phase metrics for few running VMs", func() {
-			// this tests requires at least two running VMis. To ensure this condition,
```

Comment:
Who creates the second vmi?

Reply:
its already there always in the BeforeAll

https://github.com/kubevirt/kubevirt/pull/4306/files#diff-c95f11746affb831bc43f837f8be65ad753336c090e3b181531ec00bf7636d7dL529-L531
```
			nodeName := prepareVMIForTests("")				nodeName := prepareVMIForTests("")
			// any node is fine, we don't really care, as long as we run all VMIs on it.				// any node is fine, we don't really care, as long as we run all VMIs on it.
			prepareVMIForTests(nodeName)
```

Reply:
thanks